recipes/default.rb

```diff
@@ -246,6 +246,7 @@
     db_pass:  node['mariadb']['user_librenms']['password'],
     user:     librenms_username,
     path:     librenms_homedir,
+    auto_up:  node['librenms']['auto_update']['enabled'],
```

Comment:
Do you plan to have other attributes related to auto updates? Otherwise you could just have `node['librenms']['auto_update_enabled']`.

Reply:
There are no other auto updates attributes, you are right.
Fixed.